### PR TITLE
Add documentation on how to use the tiler

### LINKED ISSUE #1957: [Task]: Add docs for the regular tiling
### What is the motivation for this task?

1. There have been some questions regarding the tiling, so it's a good idea to have this covered in the docs.

### Describe the solution you'd like

A part of docs that covers regular tiling.

It should include the following:
1. General information about tiling.
2. Which models support tiling.
3. How to setup the tiling using API.
4. How to setup the tiling using CLI.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by blaz-r**
## 📝 Description

This PR adds the docs on input tiling using regular tiler.
- 🛠️ Fixes #1957

## ✨ Changes

Select what type of change your PR is:

- [ ] 🐞 Bug fix (non-breaking change which fixes an issue)
- [ ] 🔨 Refactor (non-breaking change which refactors the code base)
- [ ] 🚀 New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] 📚 Documentation update
- [ ] 🔒 Security update

## ✅ Checklist

Before you submit your pull request, please make sure you have completed the following steps:

- [ ] 📋 I have summarized my changes in the [CHANGELOG](https://github.com/openvinotoolkit/anomalib/blob/main/CHANGELOG.md) and followed the guidelines for my type of change (skip for minor changes, documentation updates, and test enhancements).
- [x] 📚 I have made the necessary updates to the documentation (if applicable).
- [ ] 🧪 I have written tests that support my changes and prove that my fix is effective or my feature works (if applicable).

For more information about code review checklists, see the [Code Review Checklist](https://github.com/openvinotoolkit/anomalib/blob/main/docs/source/markdown/guides/developer/code_review_checklist.md).

### CODE REVIEW

File: docs/source/markdown/guides/how_to/models/input_tiling.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+# Input tiling
+
+This tutorial will show you how to tile the input to a model, using the {py:class}`Tiler <anomalib.data.utils.tiler.Tiler>`.
+
+```{warning}
+This tutorial assumes that you have already installed anomalib.
+If not, please refer to the installation section.
```
[the remark this answers is not in the document]
Comment (blaz-r) [reply docs/source/markdown/guides/how_to/models/input_tiling.md:7]: Okay, sure.

**Comment by blaz-r** [comment]
I'll address the comment tomorrow, and also fix the formatting where the checks are failing.

**Comment by samet-akcay** [comment]
> I'll address the comment tomorrow, and also fix the formatting where the checks are failing.

sure, thanks a lot for your effort

File: docs/source/markdown/guides/how_to/models/input_tiling.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (blaz-r) [reply docs/source/markdown/guides/how_to/models/input_tiling.md]: This is now fixed - there is a link directly to the installation subsection.

File: docs/source/markdown/guides/how_to/models/input_tiling.md
Comment (samet-akcay) [inline docs/source/markdown/guides/how_to/models/input_tiling.md:1]: as discussed, can we move this to `data`

Comment (blaz-r) [reply docs/source/markdown/guides/how_to/models/input_tiling.md:1]: It's moved now.

**Review by samet-akcay — APPROVED** [review]
thanks a ton for the effort!

**Pull request merged**